docs/_documentations/workingwithtemplates.md

```diff
@@ -92,11 +92,30 @@ Add your template sources to Codewind with the **Template Source Manager**.
    - **VS Code:** In the **Template Source Manager**, click **Add New**.
    - **Eclipse and IntelliJ:**
      - Right-click the connection in the Codewind Explorer view in Eclipse or the Codewind view in IntelliJ. Then, select **Manage Template Sources...**.
-     - After the **Manage Template Sources** wizard appears, click **Add...**.
+     - After the **Manage Template Sources** window appears, click **Add...**.
 2. Enter the URL and other information.
-   - **VS Code:** Enter the URL to your template source `index` file and click **Enter** to confirm.
+   - **VS Code:**
+     - Enter the URL to your template source `index` file and click **Enter** to confirm.
    - **Eclipse and IntelliJ:**
+     - In the **Add a Template Source** window, enter the template source URL.
+3. If you receive message that states the source cannot be accessed, provide authentication.
```

Comment:
For Eclipse (and probably IntelliJ) there is no automatic message that the source cannot be accessed. There is a `Test Template Source` button that they can press to see if they can access the template source.